dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -262,7 +262,7 @@ export const p2pDataModel = [
             description:
               "Use of a science-based, sector-specific Paris-compatible GHG emission reduction trajectory (Scopes 1-3) to meet climate targets and net-zero commitments represented in absolute emissions (t CO2e)",
             unit: "",
-            component: "PercentageFormField",
+            component: "NumberFormField",
```

Comment:
It was correct in the data dictionary and in the dev tools. I have no clue how this became a PercentageFormField.

Reply:
I've checked DALA-2182 of @aliabdoli77 and the field is there also a PercentageFormField. If devtools should be creating it as NumberFormField this doesn't quite seem to fit together.

Reply:
There was a bug in the data dictionary which is fixed by now.